src/intl/en/page-run-a-node.json

```diff
@@ -0,0 +1,116 @@
+  "page-run-a-node-sovereignty-title": "Sovereignty",
+  "page-run-a-node-sovereignty-preview": "Think of running a node like the next step beyond getting your own Ethereum wallet.",
+  "page-run-a-node-sovereignty-1": "An Ethereum wallet allows you to take full custody and control of your digital assets by holding the private keys to your addresses, but those keys don't tell you the current state of the blockchain, such as your wallet balance.",
+  "page-run-a-node-sovereignty-2": "By default, Ethereum wallets typically reach out to a 3rd-party node, such as Infura or Alchemy, when looking up your balances. Running your own node allows you to have your own copy of the Ethereum blockchain.",
+  "page-run-a-node-title": "Run a node",
+  "page-run-a-node-voice-your-choice-title": "Voice your choice",
+  "page-run-a-node-voice-your-choice-preview": "Don't give up control in the event of a fork.",
+  "page-run-a-node-voice-your-choice-1": "In the event of a chain fork, where two chains emerge with two different sets of rules, running your own node guarantees your ability to choose which set of rules you support. It’s up to you to upgrade to new rules and support proposed changes, or not.",
+  "page-run-a-node-voice-your-choice-2": "If you're staking ETH, running your own node allows you to chose your own client, to minimize your risk of slashing and to react to fluctuating demands of the network over time. Staking with a third party forfeits your vote on which client you think is the best choice.",
+  "page-run-a-node-what-title": "<i>What</i> does it mean to \"run a node\"?",
+  "page-run-a-node-what-1-subtitle": "Run software.",
+  "page-run-a-node-what-1-text": "Known as a \"client,\" this software downloads a copy of the Ethereum blockchain, keeps it up-to-date with new blocks and transactions, and helps others download and update their own copies.",
+  "page-run-a-node-what-2-subtitle": "With hardware.",
+  "page-run-a-node-what-2-text": "Ethereum is designed to run a client on average consumer-grade computers. You can use any personal computer, but most users opt to run their node on dedicated hardware to eliminate the performance impact on their machine and minimize node downtime.",
+  "page-run-a-node-what-3-subtitle": "While online.",
+  "page-run-a-node-what-3-text": "Running an Ethereum node may sound complicated at first, but it's merely the act of continuously running client software on a computer while connected to the internet. While offline, your node will simply be inactive until it gets back online and catches up with the latest changes.",
+  "page-run-a-node-who-title": "<i>Who</i> should run a node?",
+  "page-run-a-node-who-preview": "Nodes are not just for miners and validators. <i>Anyone</i> can run a node (you don't even need ETH).",
```

Comment:
Feel like dropping the brackets here is a bit cleaner.

![Screenshot 2022-01-07 at 23 27 38](https://user-images.githubusercontent.com/62268199/148620026-5f815206-dc30-4f66-96c7-f2b46ab542bf.png)

![Screenshot 2022-01-07 at 23 28 00](https://user-images.githubusercontent.com/62268199/148620028-632d07f4-7a0d-4855-967f-74be72fdacb1.png)


```suggestion
  "page-run-a-node-who-preview": "Nodes are not just for miners and validators. <i>Anyone</i> can run a node—you don't even need ETH.",
```

Reply:
Agreed, the ol' em dash alternative